assets/javascript/train.js

```diff
@@ -20,78 +20,121 @@ var minutesAway;
 var currentTime;
 var tableTd;
 var tableTr;
-var hourFormat;
 
-// checks current time with first train time with moment.js
-var timeCheck = function() {
-	var hourFormat = "h:mm";
-	var convertedFirstTrainTimeForm = moment(firstTrainTimeForm).format(hourFormat);
-	var convertedFrequencyForm = moment(frequencyForm).format("mm");
-	var currentTime = moment().format(hourFormat);
 
-	if (currentTime < convertedFirstTrainTimeForm) {
-		var nextArrival = convertedFirstTrainTimeForm;
-	}
-	else if (currentTime == convertedFirstTrainTimeForm) {
-		var nextArrival = "Train is here";
-	}
-	else if (currentTime > convertedFirstTrainTimeForm) {
-		while (currentTime > convertedFirstTrainTimeForm) {
-			convertedFirstTrainTimeForm.add(convertedFrequencyForm, "minutes").calendar
-			var nextArrival = convertedFirstTrainTimeForm;
-		}
-	}
-	var minutesAway = currentTime.diff(nextArrival, "minutes");
 
-};
 
 // gets user forms and gives them to firebase
-var formToFirebase = function() {
+$("#submit").on("click", function() {
 	event.preventDefault();
 	// get values from user inputed forms
 	trainForm = $("#train-form").val().trim();
 	destinationForm = $("#destination-form").val().trim();
 	firstTrainTimeForm = $("#first-train-time-form").val().trim();
 	frequencyForm = $("#frequency-form").val().trim();
-	timeCheck();
-	// Change what is saved in firebase
+	// checks current time with first train time with moment.js
+	var currentTime = moment();
+	
+	var convertedFirstTrainTimeForm = moment(firstTrainTimeForm, "HH:mm");
+	
+	
+	
+	var timeDifferent = moment().diff(moment(convertedFirstTrainTimeForm), "minutes");
+	// if loop to determine if the first train arrival is after the current time
+	if (timeDifferent > 0) {
+		var remainder = timeDifferent % frequencyForm;
+		var minutesAway = frequencyForm - remainder;
+		debugger;
+		var nextArrival = moment().add(minutesAway, "minutes").format("HH:mm");
+		debugger;
```

Comment:
I would remove these as your app is in production and these should not still be in your code, same as having unnecessary console.log's